k8s-daemonset/k8s/servicemesh.yml

```diff
@@ -0,0 +1,314 @@
+################################################################################
+# Linkerd Sevice Mesh
+#
```

Comment:
I would add a little introductory context here, just so that the file is pretty self-contained.

E.g.: "This is a basic Kubernetes config file to deploy a service mesh of Linkerd instances onto your Kubernetes cluster that is capable of handling HTTP, HTTP/2 and gRPC calls with some reasonable defaults. After deploying this, you can configure your applications to use Linkerd by setting `http_proxy` to XXX if they are HTTP servers, or by XXX

You can deploy this to your Kubernetes cluster by running `...`
"